<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>amb_monClass</class>
 <widget class="QDialog" name="amb_monClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>460</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AMB Sensors Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>scale_ruler.ico</normaloff>scale_ruler.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBoxTemp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Temperature</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelCurTemp">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Current</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelMaxTemp">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Max</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelMinTemp">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelBrdInfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="volt_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>461</width>
     <height>140</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DotLine</enum>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>22</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="pow_table">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DotLine</enum>
   </property>
   <property name="rowCount">
    <number>2</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>80</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>22</number>
   </attribute>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QGroupBox" name="groupBoxSysMon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>481</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>FPGA System Monitor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3"/>
  </widget>
  <widget class="QGroupBox" name="groupBoxIna219">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>330</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>INA219</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4"/>
  </widget>
  <widget class="QCheckBox" name="trayCheckBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Program to Tray</string>
   </property>
  </widget>
  <zorder>groupBoxIna219</zorder>
  <zorder>groupBoxSysMon</zorder>
  <zorder>groupBoxTemp</zorder>
  <zorder>labelBrdInfo</zorder>
  <zorder>volt_table</zorder>
  <zorder>pow_table</zorder>
  <zorder>trayCheckBox</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
